SamplesDashboard/SamplesDashboardTests/xunit.runner.json

```diff
@@ -1,4 +0,0 @@
-﻿{
```

Comment:
I think maybe we should put this back to reduce test failures on PR checks due to the tests in the pipeline hitting the Github API repeatedly.

Reply:
I believe the associated change should have an update in the test csproj file to ensure this is copied to the output directory as well. Or will this be included in the build directory by default?

```xml
  <ItemGroup>
    <None Update="xunit.runner.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
  </ItemGroup>
```

Reply:
Yep, you're right. https://xunit.net/docs/configuration-files.